---
layout: post
title: "shinyIGV Example"
date: 2022-03-25 12:00:00
draft: false
description: Documenting an attempt at using shinyIGV
---

## Goals 

### Introducing Rend-Seq data
The Li Lab generates end-enriched RNA sequencing (Rend-Seq) data that looks something like this:

<img width="1345" alt="image" src="https://user-images.githubusercontent.com/22749289/160170871-f560c50b-d403-4225-a63c-357a3b5ac225.png">

In the bottom pane, we can see the _mrp_ is a gene on the minus (reverse) strand of the _E. coli_ genome and _metG_ is a gene on the plus (forward) strand of the genome.

The top pane shows end-enriched reads from the plus strand. 5' (often transcription start) peaks are in blue, and 3' (often transcription stop) peaks are in red. So, for example, we can see that _metG_ has strong 3' peak at the end of the transcript, but not as strong of a 5' peak. In contrast, _mrp_ has approximately equally-strong 5' and 3' peaks.[^1] 

### Peak Calling

#### Introduction

By eye, it is easy to see end-enriched peaks in the data above. Our eyes are fairly adept at spotting peaks, but it can be a time-consuming process to annotate all peaks in the genome (Li Lab member Mandy Levine did just this with the small T4 genome and it took hundreds of hours).

For example, take a look at this genomic region:

<img width="543" alt="image" src="https://user-images.githubusercontent.com/22749289/160172166-3409d066-5c9b-49ab-bd03-710c894d7d3b.png">

It's relatively easy to see real peaks between the thiM and rcnR gene bodies. However, the signals that we see through the thiM gene body don't seem biological: those are probably sequencing artificats. Computationally, it might not be trivial to distinguish between these two. 

#### Implementation

TODO

Footnotes

[^1]: Biologically, we can justify the stronger 3' end in _metG_ various ways: for example, there's another strong blue peak close to the 3' end of the gene annotation, so maybe that's an abundant decay product). 

